renderdoc/core/core.h

```diff
@@ -65,6 +65,11 @@ struct IFrameCapturer
   virtual bool EndFrameCapture(void *dev, void *wnd) = 0;
 };
 
+// READING and EXECUTING are replay states.
+// WRITING_IDLE and WRITING_CAPFRAME are capture states.
+// WRITING is an in-between state which has context-dependent meaning.
```

Comment:
This isn't quite accurate; WRITING doesn't have a context-dependent meaning, it's just an in-between value that's useful for comparing against, we never set the state to WRITING itself.